indica/readers/abstractreader.py

```diff
@@ -544,20 +519,33 @@ def get_equilibrium(
             A dictionary containing the requested physical quantities.
 
         """
-        location_quantities = {"psi", "rmag", "zmag", "faxs", "fbnd"}
+        global_quantities = {
+            "psi",
+            "rmag",
+            "zmag",
+            "rgeo",
+            "faxs",
+            "fbnd",
+            "ipla",
+            "wp",
+            "df",
```

Comment:
This extra quantity ("df") is the reason the `PPFReader` tests are failing; `df` doesn't exist for EFIT so isn't available in the test data set.
What is `df`? Is it ST40 specific? If so, can it be moved into `_IMPLEMENTATION_QUANTITIES` for the specific reader and that will also fix the failing tests